app/Http/Controllers/CompetitionsController.php

```diff
@@ -7,7 +7,7 @@
 
 class CompetitionsController extends Controller
 {
-    private $validationRules = [
+    public static $validationRules = [
```

Comment:
Not really sure of any better way to do this 

And from an admin perspective (Chris, Hillary) having the competition form on the split page is the most convenient way to do this.